### PR TITLE
Blocks for <=, >= and != comparision.

### PR DESCRIPTION
**Pull request by AndersBillLinden**
This adds three operator blocks: ≤, ≥ and ≠.

### CODE REVIEW

**Comment by AndersBillLinden** [comment]
Must complete the branch, please wait.

**Comment by lize777** [comment]
@AndersBillLinden I think ≥, ≤ and ≠ are better.

File: src/Specs.as [hunk 1]
```diff
@@ -324,8 +324,11 @@ public class Specs {
 		["pick random %n to %n",		"r", 8, "randomFrom:to:",		1, 10],
 		["-"],
 		["%s < %s",								"b", 8, "<",					"", ""],
+		["%s ≤ %s",								"b", 8, "<=",					"", ""],
```
Comment (sclements) [inline src/Specs.as:327]: Why not use "≤" as the op string?

**Comment by thisandagain** [comment]
Resolves GH-646

**Comment by jwzimmer** [comment]
Hello again @AndersBillLinden! : )
We're not going to add these operators to scratch-flash at this time. Currently our focus for new features is Scratch 3.0; we'll consider adding the operators to our next major release.
As previously mentioned, the scratch-blocks and scratch-vm repositories are great places for open source contributions right now, and as always, the Scratch Team really appreciates your contributions! : )
Thank you!

**Pull request closed without merging**